source/tests/system/nirfmxinstr_driver_api_tests.cpp

```diff
@@ -113,6 +121,17 @@ TEST_F(NiRFmxInstrDriverApiTests, InitializeFromNIRFSA_SelfCalibrate_Succeeds)
   client::self_calibrate(stub(), session, "", 0);
 }
 
+TEST_F(NiRFmxInstrDriverApiTests, NoActiveList_GetListNames_ReturnsEmptyLists)
```

Comment:
Note: I tried adding a SpecAn list mode test but couldn't get it working on simulated hw. So we'll just rely on this minimal sanity check plus the unit test.